### DISCUSSION TITLE
Disable on mobile

### DISCUSSION THREAD

**Discussion by jacobhq** [mkosir/react-parallax-tilt#39] [Q&A] [answered]
Just wondering if there is a nice way to disable on mobile. Loving your project!

**Comment by mkosir** [comment, marked as answer]
Hey @jacobhq 👋 
glad you like the library, currently there is no functionality of detecting if tilt is on mobile device, since I think think this should be responsibility of parent/consuming application.
Tilt can be easily enabled/disabled using boolean [prop](https://github.com/mkosir/react-parallax-tilt#props) `tiltEnable`.

**Reply by jacobhq** [reply]
Ok, thank you.
